### PR TITLE
Orchest-ctl refactoring PR 5, `orchest adduser`

### PR DESCRIPTION
**Pull request by fruttasecca**
## Description

Ports `orchest adduser` to k8s. The command is not completely tested given that it requires the `auth-server` to be running, which is currently not included in our deployment.

### CODE REVIEW

File: services/orchest-ctl/app/orchest/_core.py [hunk 1]
```diff
@@ -407,3 +408,57 @@ def start():
...
+    if is_admin:
+        cmd += " --is_admin"
+    cmd = cmd.split()
+
+    resp = stream(
+        k8s_core_api.connect_get_namespaced_pod_exec,
+        pod.metadata.name,
+        config.ORCHEST_NAMESPACE,
+        command=cmd[:1],
+        args=cmd[1:],
```
Comment (yannickperrenet) [inline services/orchest-ctl/app/orchest/_core.py:458]: Given that you do `cmd.split()` I don't think `cmd[:1]` and `cmd[1:]` get the correct output.

The list would become something like: `cmd = ["python", "add_user.py", ..., "--token", "{token}", ...]`

Comment (fruttasecca) [reply services/orchest-ctl/app/orchest/_core.py:458]: Not really sure I understand what you mean, but I think I'll go ahead and just avoid this whole string and split thing in favour of just creating a list directly

Comment (yannickperrenet) [reply services/orchest-ctl/app/orchest/_core.py:458]: :+1:

**Review by yannickperrenet — APPROVED** [review]
LGTM!

**Pull request merged**